tool/tsh/common/tsh.go

```diff
@@ -2213,9 +2210,10 @@ func onLogout(cf *CLIConf) error {
 		}
 
 		fmt.Printf("Logged out all users from all proxies.\n")
-	default:
-		fmt.Printf("Specify --proxy and --user to remove keys for specific user ")
-		fmt.Printf("from a proxy or neither to log out all users from all proxies.\n")
+	case proxyHost != "" && cf.Username == "":
+		fmt.Printf("Specify --user to log out a specific user from %q or remove the --proxy flag to log out all users from all proxies.\n", proxyHost)
```

Comment:
It would be great if we can check how many users we had for each proxy and if we only have a single one, we wouldn't need to specify the username as it's filled automatically

Reply:
It would be nice, but doesn't seem to be a simple change and would add a lot of scope to this PR